Common/proto/frey.proto

```diff
@@ -51,6 +51,8 @@ service Frey {
     returns (PersonsGetPersonalInfoResponse);
   rpc FindByTenhouIds(PersonsFindByTenhouIdsPayload)
     returns (PersonsFindByTenhouIdsResponse);
+  rpc FindByMajsoulAccountId(PersonsFindByMajsoulIdsPayload)
+    returns (PersonsFindByTenhouIdsResponse);
```

Comment:
I suppose majsoul id/nickname also should be added to `PersonEx`, because online platform nicknames/ids are expected to be printed out in rating tables and game logs. Tenhou id is already there.

Reply:
fixed